lib/rules/lint-require-valid-alt-text.js

```diff
@@ -40,13 +40,20 @@ module.exports = class A11yLintAltText extends Rule {
 
         if (isImg) {
           const hasAltAttribute = AstNodeInfo.hasAttribute(node, 'alt');
+          const altValue = AstNodeInfo.elementAttributeValue(node, 'alt');
           const roleValue = AstNodeInfo.elementAttributeValue(node, 'role');
+          const srcValue = AstNodeInfo.elementAttributeValue(node, 'src');
 
           if (!hasAltAttribute) {
             this.logNode({
               message: 'img tags must have an alt attribute',
               node,
             });
+          } else if (altValue === srcValue) {
+            this.logNode({
+              message: 'the alt text must not be the same as the image source',
```

Comment:
This message doesn't really fit with the other error messages we issue here (they are sentences, but this one isn't).